src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/JsonConverter.FromJson.Null.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace System.Text.Json.Serialization
+{
+    public static partial class JsonConverter
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Given the method size, I am not sure if we should mark it with AI. How many calls to it do we have?

Reply:
There is only one call, but it's from the "main" loop so called for every null token.